src/software/ai/hl/stp/tactic/receiver/receiver_tactic.cpp

```diff
@@ -0,0 +1,67 @@
+#include "software/ai/hl/stp/tactic/receiver/receiver_tactic.h"
+
+#include "shared/constants.h"
+#include "software/ai/evaluation/calc_best_shot.h"
+#include "software/ai/hl/stp/action/move_action.h"
+#include "software/ai/hl/stp/action/stop_action.h"
+#include "software/geom/algorithms/acute_angle.h"
+#include "software/geom/algorithms/closest_point.h"
+#include "software/logger/logger.h"
+
+ReceiverTactic::ReceiverTactic()
+    : Tactic(false, {RobotCapability::Move}),
+      fsm(ReceiverFSM(std::make_shared<Timestamp>())),
```

Comment:
take a look at https://github.com/UBC-Thunderbots/Software/pull/2345 for how the constructor argument can be avoided

Reply:
turns out we don't even use the timestamp, so removed altogether